PlayList/src/com/playlist/Pause.java

```diff
@@ -0,0 +1,28 @@
+package com.playlist;
+
+import java.util.Scanner;
+
+public class Pause implements PlayerState{
+
+	
+	public void execute(AudioPlayer aPlayer) {
+		System.out.println(" || Pause.");
+		aPlayer.setState(this);
+		System.out.println("<(p)lay/(s)top>");
+		Scanner in = new Scanner(System.in);
+		String action;
+		action = in.next();
+		switch (action){
+		case "p": 
```

Comment:
[High] If you have a method for each possible action, you don't need this *switch*.Please review State pattern: http://en.wikipedia.org/wiki/State_pattern